Decred Wallet/Features/TransactionDetails/Cells/TransactionInputDetailsCell.swift

```diff
@@ -0,0 +1,86 @@
+//  TransactionInputDetailsCell.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+import UIKit
+
+class TransactionInputDetailsCell: UITableViewCell {
+    @IBOutlet weak var headerButton: UIButton!
+    @IBOutlet weak var arrowImageView: UIImageView!
+    @IBOutlet weak var debitsStack: UIStackView!
+    @IBOutlet weak var alcDebitStackHeight: NSLayoutConstraint!
+
+    var expandOrCollapse: (() -> Void)?
+    var onTxDetailValueCopied: ((_ bannerMsg: String) -> ())?
+    var isCollapsed: Bool = true
+
+    func setup(_ inputs: [TxInput], isCollapsed: Bool) {
+        self.isCollapsed = isCollapsed
+        self.headerButton.setTitle(String(format: LocalizedStrings.inputsConsumed, inputs.count), for: .normal)
```

Comment:
I'd recommend displaying this header (with the image view) as a section header view on the Transaction Details table, then the individual inputs as rows.

When creating the header view in `viewForHeaderInSection`, attach a tap gesture recognizer to perform the collapse/expand toggle. In the collapsed state, `numberOfRowsInSection` should be 0.

Same applies to the TxOutputDetailsCell. Both cells should then be renamed to singular: `TransactionInputDetailCell` and `TransactionOutputDetailCell`, as they'd basically be displaying a single input or output detail.